VitalDevices/src/main/java/io/tryvital/vitaldevices/devices/GATTMeter.kt

```diff
@@ -204,7 +212,7 @@ abstract class GATTMeter<Sample>(
                     vitalLogger.logI("Failed to emit measurement despite unlimited flow buffer size.")
                 }
             }
-            setNotificationCallback(racpCharacteristic).with { device: BluetoothDevice, data: Data ->
+            setIndicationCallback(racpCharacteristic).with { device: BluetoothDevice, data: Data ->
```

Comment:
RACP characteristic officially supports Indicate rather than Notify, so call the intended method for indications.